core/domain/stats_services.py

```diff
@@ -372,12 +367,11 @@ def update_exp_issues_for_new_exp_version(
             else:
                 playthrough.issue_customization_args['state_name']['value'] = (
                     new_state_name)
-            for idx1, action in enumerate(playthrough.playthrough_actions):
+            for idx1, action in enumerate(playthrough.actions):
```

Comment:
nit: `idx1` isn't a very helpful name. Maybe rename both `idx` variables to something like `playthrough_idx` and `action_idx`? Even the shorter: `p_idx` and `a_idx` would read better.

Reply:
That sounds nice. Done.